telegram/ext/callbackqueryhandler.py

```diff
@@ -34,10 +34,15 @@ class CallbackQueryHandler(Handler):
         pass_update_queue (optional[bool]): If the handler should be passed the
             update queue as a keyword argument called ``update_queue``. It can
             be used to insert updates. Default is ``False``
+        pass_job_queue (optional[bool]): If the callback should be passed the job queue as a
+            keyword argument called ``job_queue``. It can be used to schedule new jobs.
+            Default is ``False``
     """
```

Comment:
@jh0ker
 just shows you how much I got confused 😉
another thought:
always pass job_queue and update_queue as keyword args 
then no documentation is needed here